tools/accuracy_checker/accuracy_checker/adapters/README.md

```diff
@@ -126,6 +126,8 @@ AccuracyChecker supports following set of adapters:
 * `gaze_estimation` - converting output of gaze estimation model to `GazeVectorPrediction`.
 * `hit_ratio_adapter` - converting output NCF model to `HitRatioPrediction`.
 * `brain_tumor_segmentation` - converting output of brain tumor segmentation model to `BrainTumorSegmentationPrediction`.
+  * `make_argmax` if `True` applies argmax operation to prediction result (default - `False`)
+  * `label_order` sets mapping from output classes to dataset classes in format `class1,class2,...`
```

Comment:
```suggestion
  * `label_order`  - sets mapping from output classes to dataset classes in format `class_id1,class_id2,...`
```

Reply:
And I think you need to note which order you expect here (numeration of prediction classes should be used, right?)